README.md

```diff
@@ -33,29 +33,29 @@ Run `make run` to start the operator.
 - Get HTTP server address by running: `HTTP_SERVER=$(oc get routes flotta-operator-controller-manager -n flotta --no-headers -o=custom-columns=HOST:.spec.host)`.
 - Start yggdrasil with from the yggdrasil repository directory: 
    ```
-   sudo go run ./cmd/yggd \
-     --log-level info \
-     --protocol http \
-     --path-prefix api/flotta-management/v1 \
-     --client-id $(cat /etc/machine-id) \
-     --cert-file /etc/pki/consumer/cert.pem \
-     --key-file /etc/pki/consumer/key.pem \
-     --server $HTTP_SERVER
+    sudo ./yggd \
+    --log-level info \
+    --protocol http \
+    --path-prefix api/flotta-management/v1 \
+    --client-id $(cat /etc/machine-id) \
+    --cert-file /etc/pki/consumer/cert.pem \
+    --key-file /etc/pki/consumer/key.pem \
+    --server $HTTP_SERVER
    ```
 
 ### On minikube
 - Deploy the operator: `IMG=<image registry and tag> HOST=<host name> TARGET=k8s make deploy`.
 - Add to /etc/hosts: `<minikube ip> <host name>`.
 - Start yggdrasil with from the yggdrasil repository directory:
```

Comment:
could you change this line to:
`Start yggdrasil from the yggdrasil repository directory:`